lib/elements/reinforcement/straight_longitudinal.rb

```diff
@@ -0,0 +1,80 @@
+require 'elements/reinforcement/base'
+require 'elements/reinforcement/straight_longitudinal_layer'
+require 'errors/reinforcement/empty_layers'
+
+module Elements
+  module Reinforcement
+    class StraightLongitudinal < Base
+      attr_accessor :layers
```

Comment:
is this `attr_accessor` really needed?

Reply:
I don't think so; this intention was to expose reinforcement bars. But, I think we can expose it with a method but it is a target for other PR, don't you agree?